src/test/java/org/snomed/snowstorm/core/data/services/ConceptServiceTest.java

```diff
@@ -643,6 +643,48 @@ void testCreateDataAttribute() throws ServiceException {
 		assertEquals(PRIMITIVE, axiom.getDefinitionStatusId());
 	}
 
+	@Test
+	public void testCreateConceptWithConcreteIntValue() throws ServiceException {
+		//given
+		final Concept inConcept = new Concept("12345678910");
+		inConcept.addAxiom(
```

Comment:
@kaicode I don't think you pushed your code changes for the ConceptServiceTest; I've added in additional is-a Relationships but the _destinationId_ may be lacking context.